src/modules/file-preview-card.marko

```diff
@@ -0,0 +1,535 @@
+                        <img
+                            src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg"
+                            alt="Image"
+                        >
+                    </div>
+                    <div class="file-preview-card__menu-action">
+                        <button
+                            class="icon-btn"
+                            type="button"
+                            aria-label="Open menu"
+                        >
+                            <svg
+                                class="icon icon--16"
+                                height="16"
+                                width="16"
+                                aria-hidden="true"
+                            >
+                                <use href="#icon-overflow-vertical-16"/>
+                            </svg>
+                        </button>
+                    </div>
+                    <div class="file-preview-card__preview-action">
+                        <button class="icon-btn" type="button">
+                            <span class="file-preview-card__preview-action-text">
+                                PNG
+                            </span>
+                        </button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <highlight-code type="html">
+        <div class="file-preview-card">
+            <div class="file-preview-card__body">
+                <img
+                    src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/tb-real-square-pic.jpg"
+                    alt="Image"
+                >
+            </div>
+            <div class="file-preview-card__menu-action">
+                <button class="icon-btn" type="button" aria-label="Open menu">
+                    <svg
+                        class="icon icon--16"
+                        height="16"
+                        width="16"
+                        aria-hidden="true"
+                    >
+                        <use href="#icon-overflow-vertical-16"/>
+                    </svg>
+                </button>
+            </div>
+            <div class="file-preview-card__preview-action">
+                <button class="icon-btn" type="button">
+                    <span class="file-preview-card__preview-action-text">
+                        PNG
+                    </span>
+                </button>
+            </div>
+        </div>
+    </highlight-code>
+
+    <h3>Video</h3>
+
+    <p>For video file types, a preview is shown within the body of the card.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div style="height: 168px; width: 168px;">
+                <div class="file-preview-card">
+                    <div class="file-preview-card__body">
+                        <video src="https://storage.googleapis.com/gtv-videos-bucket/sample/ForBiggerMeltdowns.mp4"/>
```

Comment:
We have a discrepancy between the examples here. Here is a `video` tag and the sample code below shows an `img` tag.